src/Tizen.NUI/src/internal/Common/Registry.cs

```diff
@@ -82,9 +82,9 @@ internal static bool Register(BaseHandle baseHandle)
                     var target = weakReference.Target;
 
                     BaseHandle ret = target as BaseHandle;
-                    if (target == null)
+                    if (target == null || ret.Disposed || ret.IsDisposeQueued)
                     {
-                        // Special case. If WeakReference.Target is null, it might be disposed by GC. just Add forcely.
+                        // Special case. If WeakReference.Target is null, or it might be disposed by GC. just Add forcely.
```

Comment:
How about changing this description as below so as not to be confused?
```suggestion
                        // Special case. If WeakReference.Target is null or it might be disposed by GC, just add {something?} forcibly.
```
I think it'll be easier to understand if you just get rid of the dot in the middle for one sentence.
And please add something for `add` + {what} .

Reply:
Good suggestion! I change the comment as `remove previous item` instead of `add forcely`. The behavior under the comment remove the disposed item. So the keyword `remove` is more good for this cases.